apps/jenkins/jenkins/jenkins.yaml

```diff
@@ -69,6 +69,7 @@ spec:
         -Dorg.apache.commons.jelly.tags.fmt.timeZone=Europe/London
         -Djava.net.preferIPv4Stack=true
         -Dhudson.model.WorkspaceCleanupThread.retainForDays=2
+        -Dorg.jenkins.plugins.lockableresources.DISABLE_SAVE=true
```

Comment:
performance enhancement in latest release https://github.com/jenkinsci/lockable-resources-plugin#improve-performance